src/styles/_general.scss

```diff
@@ -0,0 +1,29 @@
+/*
+ * General
+ */
+
+@keyframes fade {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+body {
+  color: var(--gcds-color-grayscale-1000);
+  animation: fade 0.05s normal forwards ease-in-out;
+}
+
+.general-layout {
+  margin-top: var(--gcds-spacing-500);
+  margin-bottom: var(--gcds-spacing-500);
```

Comment:
Can you change this to `margin-block: var(--gcds-spacing-500);` instead? It targets `margin-top` and `margin-bottom` in one line and it also has better multi-language support.